osf_tests/settings.py

```diff
@@ -22,7 +22,6 @@
 
 REST_FRAMEWORK['ALLOWED_VERSIONS'] = (
         '2.0',
-        '2.0.1',
```

Comment:
Not sure. Either a bad merge or a quick trigger finger. It's still in local_travis so I'll put it back here.